test/TestSites/wwwroot/web.config

```diff
@@ -1,6 +1,6 @@
 <configuration>
   <system.webServer>
-    <httpPlatform forwardWindowsAuthToken="true" />
```

Comment:
Please comment that there's a tooling issue that prevents us from using publish-iis in the same repo as it's src.